core/jobs/batch_jobs/suggestion_edit_state_content_deletion_jobs_test.py

```diff
@@ -0,0 +1,250 @@
+                suggestion_models.SCORE_CATEGORY_DELIMITER + 'English'),
+            language_code='hi',
+            edited_by_reviewer=False
+        )
+
+        self.suggestion_4_model = self.create_model(
+            suggestion_models.GeneralSuggestionModel,
+            suggestion_type=feconf.SUGGESTION_TYPE_ADD_QUESTION,
+            target_type=feconf.ENTITY_TYPE_EXPLORATION,
+            target_id='target_4',
+            target_version_at_submission=1,
+            status='accepted',
+            author_id='author_4',
+            final_reviewer_id='reviewer_4',
+            change_cmd={},
+            score_category=(
+                suggestion_models.SCORE_TYPE_TRANSLATION +
+                suggestion_models.SCORE_CATEGORY_DELIMITER + 'English'),
+            language_code='en',
+            edited_by_reviewer=False
+        )
+
+    def test_job_deletes_suggestion_edit_state_content_model(self) -> None:
+        self.suggestion_1_model.update_timestamps()
+        self.suggestion_2_model.update_timestamps()
+        self.suggestion_3_model.update_timestamps()
+        self.suggestion_4_model.update_timestamps()
+        suggestion_models.GeneralSuggestionModel.put_multi([
+            self.suggestion_1_model,
+            self.suggestion_2_model,
+            self.suggestion_3_model,
+            self.suggestion_4_model])
+
+        queries = [(
+            'suggestion_type',
+            feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT)]
+
+        self.assertEqual(
+            len(suggestion_models.GeneralSuggestionModel.query_suggestions(
+                queries)), 2)
+
+        self.assert_job_output_is([
+            job_run_result.JobRunResult(
+                stdout='EDIT STATE CONTENT SUGGESTION SUCCESS: 2')
+        ])
+
+        self.assertEqual(
+            len(suggestion_models.GeneralSuggestionModel.query_suggestions(
+                queries)), 0)
+
+
+class AuditDeleteDeprecatedSuggestionEditStateContentModelsJobTests(
+    job_test_utils.JobTestBase):
+
+    dj = suggestion_edit_state_content_deletion_jobs
+
+    JOB_CLASS: Type[
+        dj.AuditDeprecatedSuggestionEditStateContentModelsDeletionJob
+    ] = dj.AuditDeprecatedSuggestionEditStateContentModelsDeletionJob
+
+    def setUp(self) -> None:
+        super().setUp()
+
+        self.suggestion_1_model = self.create_model(
+            suggestion_models.GeneralSuggestionModel,
+            suggestion_type=feconf.SUGGESTION_TYPE_EDIT_STATE_CONTENT,
+            target_type=feconf.ENTITY_TYPE_EXPLORATION,
+            target_id='target_1',
+            target_version_at_submission=1,
+            status='accepted',
+            author_id='author_1',
+            final_reviewer_id='reviewer_1',
+            change_cmd={},
+            score_category=(
+                suggestion_models.SCORE_TYPE_TRANSLATION +
+                suggestion_models.SCORE_CATEGORY_DELIMITER + 'English'),
+            language_code=None
+        )
```

Comment:
I think you can maybe avoid the repetition of the model creation by having a function at the top that can be used by both the classes?

Reply:
Done